.github/workflows/performance-tests.yml

```diff
@@ -52,164 +44,219 @@ jobs:
       - uses: stackrox/actions/infra/install-infractl@v1
       - uses: "google-github-actions/auth@v1"
         with:
-          # TODO: Add new service account
-          credentials_json: "${{ secrets.GCP_RELEASE_AUTOMATION_SA }}"
+          credentials_json: "${{ secrets.GCP_PERFORMANCE_TESTER_CI_SA }}"
       - name: "Set up Cloud SDK"
         uses: "google-github-actions/setup-gcloud@v1"
         with:
           install_components: "gke-gcloud-auth-plugin"
-      - name: Download artifacts
+
+      - name: Prepare test environment
         run: |
-          infractl artifacts "${NAME//./-}" --download-dir artifacts >> "$GITHUB_STEP_SUMMARY"
+          infractl artifacts "${NAME//./-}" --download-dir artifacts > /dev/null
 
           # Debug if we can access cluster and if central is running.
           kubectl get pods --namespace stackrox
 
           # Required for k6 tests to run.
           echo "HOST=$(cat artifacts/url)" >> "$GITHUB_ENV"
           echo "ROX_ADMIN_PASSWORD=$(cat artifacts/admin-password)" >> "$GITHUB_ENV"
+          echo "::add-mask::$(cat artifacts/admin-password)"
+
+          # Create result directory
+          mkdir -p performance-results
```

Comment:
Important, because K6 needs this directory to place the results.

Reply:
In which way we run the performance tests locally, this directory should also be created...